jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/vo/BlobServiceVH.java

```diff
@@ -6,6 +6,15 @@ public class BlobServiceVH extends AzureVH{
     private String name;
     private String type;
     private Map<String, Object> propertiesMap;
+    private Map<String, String> tags;
+
+    public Map<String, String> getTags() {
+        return tags;
+    }
+
+    public void setTags(Map<String, String> tags) {
+        this.tags = tags;
+    }
```

Comment:
To reduce duplicate can we move the tags, getter and setter to AzureVH.

Reply:
As discussed the type is Map<String, Object> tags in some collectors because of which it cannot be moved to  GCPVH